wire-ios-sync-engine/Source/Synchronization/SyncAgent.swift

```diff
@@ -68,6 +68,7 @@ final class SyncAgent: NSObject, SyncAgentProtocol {
     private let initialSyncTaskManager = NonReentrantTaskManager()
     private var incrementalSyncToken: IncrementalSync.Token?
     private var ongoingSyncTask: Task<Void, Never>?
+    private let conversationUpdatesGenerator: ConversationUpdatesGeneratorProtocol
```

Comment:
issue: This won't scale well as we add more generators, so I think this would be better outside of the `SyncAgent` and instead react to the delegate methods (possibly adding another for suspend) to start and stop the monitoring.